packages/deta/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "test:deno": "echo \"Error: no tests found\"",
     "prebuild": "rimraf dist",
-    "build": "deno2node tsconfig.cjs.json && deno2node tsconfig.esm.json && pnpm postbuild",
+    "build": "tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && pnpm postbuild",
```

Comment:
I assume this is just for testing? Am I right that you would migrate to JSR and then skip the whole build process for publishing?